beets/autotag/hooks.py

```diff
@@ -66,6 +66,7 @@ class AlbumInfo(object):
     - ``albumstatus``: MusicBrainz release status (Official, etc.)
     - ``media``: delivery mechanism (Vinyl, etc.)
     - ``albumdisambig``: MusicBrainz release disambiguation comment
+    - ``releasegroupdisambig``: MusicBrainz release group disambiguation comment
```

Comment:
Travis has the most nitpicky style nitpick ever on this line: https://github.com/beetbox/beets/pull/3025/files